sycl/source/detail/scheduler/commands.cpp

```diff
@@ -2033,6 +2038,22 @@ static pi_result SetKernelParamsAndLaunch(
     }
   };
 
+  for (auto &arg : Args) {
+    if (arg.MType == kernel_param_kind_t::kind_accessor) {
```

Comment:
What about converting this into an early exist to reduce amount of indentation levels?